server/controllers/manage/premises/bedsController.ts

```diff
@@ -12,19 +24,29 @@ export default class BedsController {
       const {
         user: { token },
         params: { premisesId },
+        query,
       } = req
 
+      const filterCharacteristics = makeArrayOfType<Cas1SpaceBookingCharacteristic>(query.characteristics)
+
       const [premises, beds] = await Promise.all([
         this.premisesService.find(token, premisesId),
         this.premisesService.getBeds(token, premisesId),
       ])
 
+      const filteredBeds = filterBedsByCharacteristics(beds, filterCharacteristics as Array<Cas1SpaceCharacteristic>)
```

Comment:
Can `filterCharacteristics` be instantiated directly as `makeArrayOfType<Cas1SpaceCharacteristic>(...)` to avoid this casting? It seems at this Premises manage level we shouldn't need to reference `Cas1SpaceBookingCharacteristic`.